Testing/ORKTest/ORKTestTests/ORKJSONSerializationTests.m

```diff
@@ -373,12 +378,13 @@ - (void)testORKSerialization {
                                                      [ORKStepNavigationRule class],     // abstract base class
                                                      [ORKSkipStepNavigationRule class],     // abstract base class
                                                      [ORKStepModifier class],     // abstract base class
-                                                     [ORKPredicateSkipStepNavigationRule class],     // NSPredicate doesn't yet support JSON serialzation
-                                                     [ORKKeyValueStepModifier class],     // NSPredicate doesn't yet support JSON serialzation
-                                                     [ORKCollector class], // ORKCollector doesn't support JSON serialzation
+                                                     [ORKPredicateSkipStepNavigationRule class],     // NSPredicate doesn't yet support JSON serialization
+                                                     [ORKKeyValueStepModifier class],     // NSPredicate doesn't yet support JSON serialization
+                                                     [ORKCollector class], // ORKCollector doesn't support JSON serialization
                                                      [ORKHealthCollector class],
                                                      [ORKHealthCorrelationCollector class],
-                                                     [ORKMotionActivityCollector class]
+                                                     [ORKMotionActivityCollector class],
+                                                     [ORKShoulderRangeOfMotionStep class],
```

Comment:
Shoulder range of motion step doesn't serialize properly for some reason; excluded for now. -limbOption isn't available if encoding from the ORKRangeOfMotionStep subclass itself.